### PR TITLE
Plugins - Minor util script

### PR DESCRIPTION
**Pull request by wild-endeavor**
# TL;DR
This adds a small bash script to help manage the new set of plugins

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [ ] Unit tests added
 - [x] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
* Added bash script
* Ran `make requirements` which got rid of the weird `extra="all"` in the requirements file that was giving us problems earlier.

### CODE REVIEW

File: plugins/run_all_plugins.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+#Usage
+#  run_all_plugins.sh $@
+#
+# This script is intended to be run from the base cookbook_se directory. It will go through all sub-folders in the
```
Comment (katrogan) [inline plugins/run_all_plugins.sh:6]: uhh cookbook_se?

Comment (wild-endeavor) [reply plugins/run_all_plugins.sh]: oops updated

File: plugins/Makefile [hunk 2]
```diff
@@ -5,3 +5,8 @@ test:
 .PHONY: install-all-dev
 install-all-dev:
 	pip install -e .
+
+.PHONY: build_all_plugins
+build_all_plugins:
```
Comment (kumare3) [inline plugins/Makefile:10]: how about run tests for all plugins?

Comment (wild-endeavor) [reply plugins/Makefile:10]: can i do it separately?  it's currently run by the top level Makefile.

**Comment by wild-endeavor** [comment]
tests failed due to sqlite task timing out.

**Review by kumare3 — APPROVED** [review]

**Pull request merged**